src/NuGet.Core/NuGet.Commands/RestoreCommand.cs

```diff
@@ -1159,19 +1159,20 @@ private RuntimeGraph LoadRuntimeGraphCore(LocalPackageInfo package)
             }
         }
 
-        private async Task InstallPackageAsync(RemoteMatch installItem, string packagesDirectory, CancellationToken token)
+        private Task InstallPackageAsync(RemoteMatch installItem, string packagesDirectory, CancellationToken token)
```

Comment:
It seemed unusual. I wondered what motivation was behind the change. I think we'd better stick to convention